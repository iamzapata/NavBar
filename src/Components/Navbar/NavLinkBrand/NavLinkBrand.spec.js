import React from "react"
import { render, cleanup } from "react-testing-library"
import { Provider } from "react-redux"
import configureMockStore from "redux-mock-store"
import { BrowserRouter as Router } from "react-router-dom"
import NavLinBrand from "./index"

const mockStore = configureMockStore()
const store = mockStore({ Theme: { themeName: "Pinterest" } })

describe("<NavLinBrand />", () => {
  afterEach(cleanup)
  it("Should render self", () => {
    const { container } = render(
      <Provider store={store}>
        <Router>
          <NavLinBrand />
        </Router>
      </Provider>
    )
    expect(container.firstChild).toBeTruthy()
    expect(container.firstChild).toMatchSnapshot()
  })
})
