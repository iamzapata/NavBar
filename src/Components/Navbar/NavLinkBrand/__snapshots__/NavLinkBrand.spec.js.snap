// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<NavLinBrand /> Should render self 1`] = `
<a
  aria-current="page"
  class="NavLInkBrand navbar-item active"
  href="/"
>
  <img
    alt="Pinterest"
    src="test-file-stub"
  />
  <span
    class="BrandName is-size-5"
  >
    Pinterest
  </span>
</a>
`;
